overrides.d.ts

```diff
@@ -2,3 +2,8 @@
 // errors.
 // TODO: create issue in Gatsby repo
 declare module "gatsby-plugin-image"
+
+declare module "*.yaml" {
+  const content: { [key: string]: any }
+  export default content
+}
```

Comment:
I think this is fine but I don't love it. I encourage you to define it a bit better (not using `any` and define it for each yaml file).

```ts
declare module "*developer-docs-links.yaml" {
  export interface DeveloperDocsLink {
    id: string
    to: string
    path: string
    description: string
    items: Array<DeveloperDocsLink>
  }
  const content: Array<DeveloperDocsLink>
  export default content
}

// then in DocsNav.tsx  you can use it like this
import docLinks, { DeveloperDocsLink } from "../data/developer-docs-links.yaml"
....
const getDocs = (links: Array<DeveloperDocsLink>): void => {
```